tests/ci/test_harness_coherence_measure.py

```
@@ -0,0 +1,886 @@
+
+        # Second: with plugins
+        cdaa_path = tmp_path / "cdaa.json"
+        cdaa_path.write_text(json.dumps(cdaa_sample), encoding="utf-8")
+        he_path = tmp_path / "he.json"
+        he_path.write_text(json.dumps(harness_evolution_sample), encoding="utf-8")
+        report_with = build_report()
+
+        # Criteria scores are identical
+        for c_without, c_with in zip(
+            report_without["criteria"], report_with["criteria"]
+        ):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)